mirgecom/initializers.py

```diff
@@ -370,6 +465,76 @@ def exact_rhs(self, discr, q, t=0.0):
         return flat_obj_array(massrhs, energyrhs, momrhs)
 
 
+class AcousticPulse:
+    r"""
+    Create an N-dimensional Gaussian acoustic pulse.
+
+    The Gaussian pulse is defined by:
+
+    .. math::
+
+        {\rho}E(\vec{r}) = {\rho}E + amplitude * G(\vec{r})\\
+        G(\vec{r}) = \exp^{-(\frac{(\vec{r}-\vec{r_0})}{\sqrt{2}w})^{2}}
+
+    Where :math:`\vec{r}` are the nodal coordinates, and :math:`\vec{r_0}`,
+    :math:`amplitude`, and :math:`w`, are the the user-specified pulse location,
+    amplitude, and width, respectively.
+
+    .. automethod:: __init__
+    .. automethod:: __call__
+    """
+
+    def __init__(self, numdim=1, amplitude=1,
+                 center=None, width=1):
+        r"""
+        Initialize acoustic pulse parameters.
+
+        Parameters
+        ----------
+        numdim : int
```

Comment:
Be consistent. Is there a space in front of the colon or not? This and the following docstring are mutually inconsistent.